src/System.Net.WebSockets.Client/tests/ClientWebSocketUnitTest.cs

```diff
@@ -57,17 +57,21 @@ public void CloseAsync_CreateAndClose_ThrowsInvalidOperationException()
             }
         }
 
-        [ActiveIssue(20132, TargetFrameworkMonikers.Uap)]
         [OuterLoop] // TODO: Issue #11345
         [ConditionalFact(nameof(WebSocketsSupported))]
-        public void CloseAsync_CreateAndCloseOutput_ThrowsInvalidOperationExceptionWithMessage()
+        public async Task CloseAsync_CreateAndCloseOutput_ThrowsInvalidOperationExceptionWithMessage()
         {
             using (var cws = new ClientWebSocket())
             {
-                AssertExtensions.Throws<InvalidOperationException>(
-                    () =>
-                    cws.CloseOutputAsync(WebSocketCloseStatus.Empty, "", new CancellationToken()).GetAwaiter().GetResult(),
-                    ResourceHelper.GetExceptionMessage("net_WebSockets_NotConnected"));
+                var exception = await Assert.ThrowsAsync<InvalidOperationException>(
+                    () => cws.CloseOutputAsync(WebSocketCloseStatus.Empty, "", new CancellationToken()));
+
+                // The .Net Native toolchain optimizes away exception messages.
+                if (!PlatformDetection.IsNetNative)
```

Comment:
>PlatformDetection.IsUapAot doesn't exist :(

@safern @danmosemsft please advise.  We use TargetFrameworkMonikers.Uap or .UapAot for ActiveIssue and SkipOnTargetFramework attributes.   Is PlatformDetection.IsNetNative stable enough to use?  Will it always be "UAPAOT" even when/if the test runner changes to AppX/app-container?